### ISSUE TITLE
Cannot sort threads

### ISSUE THREAD

**Issue by NicolasLM** [nirs/pytest-threadleak#1]
With Python 3.6.1 on Linux when multiple threads leak:
```
>   pytest.fail("Test leaked (%s)" % sorted(leaked_threads))
E   TypeError: '<' not supported between instances of 'Thread' and 'Thread'
```

Removing the `sorted` fixes the issue.

**Comment by nirs** [comment]
Thanks Nicolas! do you want to post a patch?

**Comment by nirs** [comment]
Fixed in version 0.2.0.
